doc/getting_started/_index.md

```diff
@@ -63,6 +63,15 @@ On Ubuntu 18.04, the required packages can be installed with the following comma
 
 {{< pkgmgr_cmd "apt" >}}
 
+Check that you have version 9 or higher of `gcc` installed by:
+```console
+$ gcc --version
+```
+If your version of `gcc` is lower, you can try installing the higher version by:
+```console
+$ sudo apt install gcc-9
+```
```

Comment:
@marnovandermaas  I made a pull request into your forked OpenTitan repository with some changes according to the link mentioned by @m-temp. In the GitHub notes (https://docs.opentitan.org/doc/ug/github_notes/#creating-updates-on-top-of-a-pending-pull-request) it says this would be the correct way to update a pending pull request.
In case it's too much feel free to edit it.

Reply:
Ok, that should be updated now.